pegasus/sites.v3/code.org/views/plc_plp_nav.haml

```diff
@@ -1,7 +1,7 @@
 #about_nav
   %span.about_list
-    -current_url = '/educate/plc/plp'
-    -{'Apply to be a Partner'=>'plp-application', 'Terms of Partnership'=>'plp-terms', 'Where we need partners'=> current_url+'#partners', 'FAQ'=>'plp-faq'}.each_pair do |label,url|
+    -current_url = '/educate/professional-learning-partner'
```

Comment:
Charing and I discussed offline but just for documenting purposes. The nav isn't dynamic because it has two levels of hierarchy. If we used `current_url = request.path_info` then that would break the in-page links like `#partner`. If we were to change it to be dynamic, there would have to be special cases for in-page links to get the correct current_url. Either way, this would also have to be updated if the pages moved again. So we'll stick with this for now so it's obvious what the expected page is and can be search/replaced when reorganized.